includes/javascript/sliders.js

```diff
@@ -0,0 +1,45 @@
+jQuery( document ).ready(function($) {
+ 
+	if( $("#schoolQuestForm").length ){
+		console.log("SCHOOL");
+		 var x = 0;
+		 var y = 1;
+		 var school = ".schoolSlider" + y;
+		 while(x == 0){
+			if ( $( school ).length ) {
+		 
+				$(school).slider({ min: 0, max: 5, value: 0, focus: true });
+
+			}
+			else{
+				x = 1;
+			}
+			y = y + 1;
+			school = ".schoolSlider" + y;
+			 console.log(school);
+		 }
+	}
+	
+	if( $("#companyQuestForm").length ){
+		 var x = 0;
+		 var y = 1;
+		 var companyDS = ".companyDS" + y;
+		 var companySS = ".companySS" + y;
+		 
+		 while(x == 0){
+			if ( $( companyDS ).length ) {
+		 
+				$(companyDS).slider({ min: 0, max: 5, value: [0, 5], focus: true });
+				$(companySS).slider({ min: 0, max: 3, value: 0, focus: true });
+
+			}
+			else{
+				x = 1;
+			}
+			y = y + 1;
+			companyDS = ".companyDS" + y;
+			companySS = ".companySS" + y;
+			console.log(school);
```

Comment:
What have you intended to print out here? Variable `school` is not defined in this scope.

Reply:
Right, sorry, just some leftover test code